core/vidl/vidl_ffmpeg_ostream.cxx

```diff
@@ -30,8 +30,10 @@ extern "C" {
 # include "vidl_ffmpeg_ostream_v4.txx"
 #elif LIBAVCODEC_BUILD < ((54<<16)+(23<<8)+100)  // before ver 54.23.100
 # include "vidl_ffmpeg_ostream_v0.9.txx"
-#else
+#elif LIBAVCODEC_BUILD < ((56<<16)+(0<<8)+100)  // before ver 56.0.100
```

Comment:
Ideally what we want here is the last version that is supported by the vidl_ffmpeg_ostream_v0.11.txx implementation.  However, since there is a huge range of version that we don't support, it's broken either way for those in between versions.  So it's not critical that this is addressed in this branch. 

A better approach in the future would be to check for those unsupported version and throw an error.  I think Kitware has code to do that that we will contribute in the future.